Rubberduck.Parsing/Grammar/VBAParser.g4

```diff
@@ -289,16 +291,17 @@ defType :
 // singleLetter must appear at the end to prevent premature bailout
 letterSpec : universalLetterRange | letterRange | singleLetter;
 
-// we're purpusefully not actually making sure this is an ASCII letter
-// that'd be too much of a hassle with the lexer to get working
-singleLetter : {_input.Lt(1).Text.Length == 1}? IDENTIFIER;
+singleLetter : {_input.Lt(1).Text.Length == 1 && Regex.Match(_input.Lt(1).Text, @"[a-zA-Z]").Success}? IDENTIFIER;
```

Comment:
I think this needs a explanatory comment why we are resorting to .NET's regex here, instead of using ANTLR's quirky regex.

Reply:
ANTLR regexes are for Lexer rules. We can't use them in the Parser. Also we can't use ANTLR Regexes in semantic predicates, because those need to be written in the target language...

Reply:
The more interesting comment would be regarding why we're not using `Char.IsLetter`, but reading the docs of that method makes it clear :/